Cartfile.resolved

```diff
@@ -0,0 +1 @@
+github "GetStream/stream-chat-swift" "4.6.0"
```

Comment:
Based on my experience, it is better not to commit this file. Seems like it makes dependency resolution more complex in some cases

Reply:
Done.